minibear/src/main/java/frc/robot/NavX/AHRS.java

```diff
@@ -0,0 +1,1652 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
need to leave readme in this doc about where this code came from:
// Use navx2 workaround from https://github.com/Thunderstamps/navx2workaround, 
// rather than com.kauailabs.navx.frc.AHRS.  To fix MXP/SPI discconnect issues.